src/com/owncloud/android/lib/common/operations/RemoteOperation.java

```diff
@@ -90,23 +96,29 @@
      * 
      * Do not call this method from the main thread.
      * 
-     * This method should be used whenever an ownCloud account is available, instead of {@link #execute(OwnCloudClient)}. 
+     * This method should be used whenever an ownCloud account is available, instead of
+     * {@link #execute(OwnCloudClient, java.lang.String)}.
      * 
-     * @param account   ownCloud account in remote ownCloud server to reach during the execution of the operation.
+     * @param account   ownCloud account in remote ownCloud server to reach during the
+     *                  execution of the operation.
      * @param context   Android context for the component calling the method.
+     * @param userAgent userAgent string
      * @return          Result of the operation.
      */
-    public RemoteOperationResult execute(Account account, Context context) {
+    public RemoteOperationResult execute(Account account, Context context, String userAgent) {
```

Comment:
We can't do this, need to find other solution. If the user agent is set in the OC client, we shouldn't need to include it every execute() call; that redundancy in parameters should not be needed.